DATA/UNIVERSE/SYSTEMS/Bw10/Bw10.ini

```diff
@@ -1006,106 +1007,178 @@ ids_info = 65921
 
 [Object]
 nickname = Bw10_Trade_Lane_Ring_Construct_B_1

```

Comment:
Nit-picking of doom, but can they be named like regular rings without the Construct part?

Reply:
I can rename them. I was trying to keep all of the content names unchanged, but easy to fix.